pkg/core/blockchain.go

```diff
@@ -504,6 +504,15 @@ func (bc *Blockchain) persist() error {
 			"blockHeight":     bHeight,
 			"took":            time.Since(start),
 		}).Info("blockchain persist completed")
+
+		BCMetrics := &BlockChainMetrics{}
+		BCMetrics.Update(
```

Comment:
This looks weird. `BlockChainMetrics` has no real meaning at all. I guess it's better to have one singleton package-level structure.

Reply:
updated